design2.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link href="./output.css" rel="stylesheet">
+    </head>
+    <body class="bg-gradient-to-br from-[#3f63de] to-[#8c4ec5]">
+        <div class="flex justify-center items-center h-screen text-white gap-4">
+        <span class="text-xl font-semibold">First name</span>
+        <span class="border border-black rounded p-2 pr-32 text-xl bg-dark/15 border-opacity-40">Pedro Duarte</span></div>

```

Comment:
1. Die Border Opactity kann genau so wie die Background Opacity über `color/opacity` gesetzt werden. Ist kürzer -> ist schöner.
2. Du erzielst die Breite des Elements hier unkonventionell über einen sehr großen rechten Padding. Das Ergebnis stimmt zwar, aber die Idee dahinter ist nicht ganz richtig. Du willst hier nicht explizit einen großen, rechten Padding haben, sondern eben ein Feld in einer bestimmten Breite. Also definiere das auch gerne so: Nämlich über eine feste Breite.
3. Du verwendest hier aktuell ein `span`. Das Element ist allerdings ein `input` Element. "Pedro Duarte" ist dabei nur ein Beispielinhalt, der händisch eingetragen werden kann.